roles/openshift_excluder/tasks/status.yml

```diff
@@ -17,37 +17,37 @@
   register: docker_excluder_installed
   failed_when: false
 
-- name: Update to latest excluder packages
-  package:
-    name: "{{ openshift.common.service_type }}-excluder"
-    state: latest
-  when:
-  - "{{ openshift_excluder_installed.installed_versions | default([]) | length > 0 }}"
-  - not openshift.common.is_containerized | bool
-
-- name: Update to the latest docker-excluder packages
-  package:
-    name: "{{ openshift.common.service_type }}-docker-excluder"
-    state: latest
-  when:
-  - "{{ docker_excluder_installed.installed_versions | default([]) | length > 0 }}"
-  - not openshift.common.is_containerized | bool
+- block:
+  - name: Update excluder packages if requested
```

Comment:
@sdodson I still think we should avoid upgrading the excluder outside of initial install, with the exception of ensuring that we have the minimum version required for providing the status output. If someone is running playbooks/byo/openshift-cluster/config.yml for their ongoing configuration management, I'm not sure we'd want to mutate package versions unless absolutely necessary.

It's probably less important today where we have plenty of other tasks that lack idempotency, but it could be an unwelcome surprise.